packages/fxa-settings/README.md

```diff
@@ -51,6 +51,23 @@ FxA Settings communicates primarily with the FxA GraphQL API through use of [Apo
 
 While most API calls can be performed with GQL, there are a few calls that must directly communicate with the auth server (for now) for security reasons. To make REST API calls to the auth server, use the `useAuth` and `useAwait` hooks.
 
+### Error handling
+
+All errors that can occur as a result of a network action should be handled. As a general rule of thumb validation errors should be displayed the user while all others should be reported to Sentry unless they are know to occur and intentionally ignored.
```

Comment:
Add to table of contents? 😀 Also we may want to title this "GQL error handling" since REST errors (whenever we get there) won't be applicable.

For this paragraph, I think we could expound upon what's meant for "network action," something like this maybe? (double check me for accuracy 😉)

"When an error occurs from a query or mutation, the Apollo Client GQL response distinguishes between two kinds of errors on the `error` object - `graphQLErrors`, an array of errors provided by a GQL server-side resolver, and `networkError`, an error that is thrown outside of a resolver. We don't need to send `graphQLErrors` to Sentry because they're logged by the `fxa-gql-api` package, but a `networkError` should be logged because it means the query was rejected and no data was returned, such as if Apollo Client failed to connect to the GQL endpoint.

All errors should be handled. As a general rule of thumb, validation errors (which are in the `graphQLErrors` array) should be displayed the user, while all others should be reported to Sentry unless they are know to occur and should be intentionally ignored."

Reply:
Oh also, Ben had mentioned validation errors aren't reported to Sentry but other errors on `graphQLErrors` are so maybe we add something about that. I'm not sure if we normally log "invalid code" and such in Sentry, it seems like that wouldn't be useful...

Reply:
Thanks for the updated copy. I think that works, but I'll keep the section about using the useMutation hook if thats cool. I also don't think we need to log non-GQL validation errors.

Reply:
Oh yeah absolutely, I just meant for the first paragraph. I think "When performing a GraphQL mutation" etc. is great.